common/djangoapps/dark_lang/views.py

```diff
@@ -0,0 +1,153 @@
+"""
+Views file for the Darklang Django App
+"""
+from openedx.core.lib.api.view_utils import view_auth_classes
+from django.views.generic.base import View
+from django.contrib.auth.decorators import login_required
+from django.utils.decorators import method_decorator
+
+from edxmako.shortcuts import render_to_response
+
+from openedx.core.djangoapps.user_api.preferences.api import (
+    delete_user_preference, get_user_preference, set_user_preference
+)
+from lang_pref import LANGUAGE_KEY
+
+from django.utils.translation import LANGUAGE_SESSION_KEY
+from django.utils.translation import ugettext as _
+
+from dark_lang import DARK_LANGUAGE_KEY
+from dark_lang.models import DarkLangConfig
```

Comment:
I will work on this, I am trying to learn the coding style quickly, so thank you for pointing this out.

How do I know the difference between a "Core Django import" and a "Third party Django import" ?